lib/matplotlib/backend_bases.py

```diff
@@ -2331,6 +2331,10 @@ def on_press(event):
             cid = canvas.mpl_connect('button_press_event', on_press)
 
         """
+        if s == 'idle_event':
+            warnings.warn("idle_event is only implemented for the wx backend, "
+                          "and will be removed in matplotlib 2.0. Use the "
```

Comment:
Can you bump this to 2.1?  I want to hold the line that the only API changes in 2.0 will be style changes.